core/chains/evm/txmgr/orm_test.go

```diff
@@ -1552,3 +1554,48 @@ func TestORM_PruneUnstartedEthTxQueue(t *testing.T) {
 		assert.Equal(t, int64(2), n)
 	})
 }
+
+func TestORM_ConvertArgs(t *testing.T) {
+	t.Parallel()
+
+	var compareTo pg.QOpt = func(*pg.Q) {}
+	var opt any = pg.WithParentCtx(context.Background())
+
+	t.Run("ToQOpt succeeds any -> QOpt", func(t *testing.T) {
+		qopt, err := txmgr.ToQOpt(opt)
+		require.NoError(t, err)
+		require.IsType(t, compareTo, qopt)
+	})
+
+	t.Run("ToQOpt fails any -> QOpt", func(t *testing.T) {
+		var err error
+		// pass []any instead of any
+		_, err = txmgr.ToQOpt([]any{opt})
+		require.Error(t, err)
+		_, err = txmgr.ToQOpt(8)
+		require.Error(t, err)
+	})
+
+	t.Run("ToQOpts succeeds []any -> []QOpt", func(t *testing.T) {
+		qopts, err := txmgr.ToQOpts([]any{opt})
+		require.NoError(t, err)
+		require.IsType(t, compareTo, qopts[0])
+	})
+
+	t.Run("ToQOpts fails []any -> []QOpt", func(t *testing.T) {
+		var opts = make([]any, 1)
+		opts[0] = 42
+		_, err := txmgr.ToQOpts(opts)
+		require.Error(t, err)
+	})
+
+	t.Run("can convert variadic function", func(t *testing.T) {
```

Comment:
this is good to know. You can pass in paramters individually to a variadic function and Go will cast as `any`. However it won't work if you expand a slice/array (see above)